qtodotxt/lib/file.py

```diff
@@ -50,16 +51,19 @@ def __str__(self):
     __repr__ = __str__
 
     def load(self, filename):
-
+        lines = ""
         try:
             with open(filename, 'rt', encoding='utf-8') as fd:
                 lines = fd.readlines()
         except FileNotFoundError:
-            raise ErrorLoadingFile("Trying to load a non-existing file: '{}".format(filename))
+            ErrorLoadingFile("Trying to load a non-existing file: '{}".format(filename))
+            return False
         except IOError as ex:                # deprecated since Python 3.3, it would be OSError for =>3.3-support
-            raise ErrorLoadingFile(str(ex))
+            ErrorLoadingFile(str(ex))
+            return False
```

Comment:
what error? In am 100% sure that raising and catching  an exception works on Windows too. Show me the stack trace. We may get different exceptions on Windows but we can solve that by catching more exceptions at lower level and raising our custom exception